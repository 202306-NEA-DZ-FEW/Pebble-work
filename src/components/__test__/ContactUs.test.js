import React from "react";
import renderer from "react-test-renderer";

import ContactForm from "../ContactUs";

test("ContactForm snapshot", () => {
    const component = renderer.create(<ContactForm />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
