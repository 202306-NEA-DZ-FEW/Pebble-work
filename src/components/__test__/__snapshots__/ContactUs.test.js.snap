// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactForm snapshot 1`] = `
<div
  className="formHeight text-black sm:w-[450px] w-[90vw] h-[450px] flex flex-col items-center mx-auto gap-4"
>
  <h1
    className="text-[26px]"
  >
    contactForm.title
  </h1>
  <form
    className="jsx-f939cb32bf8ff4c1 h-[40vh]"
    onSubmit={[Function]}
  >
    <div
      className="jsx-f939cb32bf8ff4c1 input-row"
    >
      <input
        className="jsx-f939cb32bf8ff4c1 "
        id="fname"
        name="firstname"
        onChange={[Function]}
        placeholder="contactForm.firstNamePlaceholder"
        style={
          {
            "background": "var(--fill-white, #FFF)",
            "borderRadius": "4px",
          }
        }
        type="text"
        value=""
      />
      <input
        className="jsx-f939cb32bf8ff4c1 "
        id="lname"
        name="lastname"
        onChange={[Function]}
        placeholder="contactForm.lastNamePlaceholder"
        style={
          {
            "background": "var(--fill-white, #FFF)",
            "borderRadius": "4px",
          }
        }
        type="text"
        value=""
      />
    </div>
    <input
      className="jsx-f939cb32bf8ff4c1"
      id="email"
      name="email"
      onChange={[Function]}
      placeholder="contactForm.emailPlaceholder"
      style={
        {
          "background": "var(--fill-white, #FFF)",
          "borderRadius": "4px",
        }
      }
      type="email"
      value=""
    />
    <br
      className="jsx-f939cb32bf8ff4c1"
    />
    <input
      className="jsx-f939cb32bf8ff4c1"
      id="phone"
      name="phone"
      onChange={[Function]}
      placeholder="contactForm.phonePlaceholder"
      style={
        {
          "background": "var(--fill-white, #FFF)",
          "borderRadius": "4px",
        }
      }
      type="tel"
      value=""
    />
    <br
      className="jsx-f939cb32bf8ff4c1"
    />
    <textarea
      className="jsx-f939cb32bf8ff4c1 "
      id="message"
      name="message"
      onChange={[Function]}
      placeholder="contactForm.messagePlaceholder"
      style={
        {
          "background": "var(--fill-white, #FFF)",
          "borderRadius": "4px",
        }
      }
      value=""
    />
    <br
      className="jsx-f939cb32bf8ff4c1"
    />
    <input
      className="jsx-f939cb32bf8ff4c1"
      type="submit"
      value="contactForm.submitButton"
    />
  </form>
</div>
`;
